landlab/components/uniform_precip/generate_uniform_precip.py

```diff
@@ -13,7 +13,8 @@
 
 import random
 import numpy as np
-from landlab import Component
+from landlab import Component, ModelGrid
+from six import next
 
 
```

Comment:
Sorry! Hadn't realised you were out of the office. I think the approach we've had until now is to assume standard units are m and yr. In lots of cases, it doesn't really matter, as long as the user checks all the unit conversions. _However,_ there definitely are components where this isn't the case, and things need to be geared correctly. This tends to happen anywhere where a component takes both a rapid flux as an input and produces a slow change as an output. In these cases, typically discharge [m**3/s], but dz/dt [m/y]. I think that because of these cases, we've basically just asserted `m` and `y` everywhere, to be on the safe side. The most obvious cases where the units could be different I think are generally flagged in the description.

Let's talk this over on a call, as you say.